### PR TITLE
FAQs for Available Source License

### PR DESCRIPTION
**Pull request by justinegeffen**
Added Available Source License section. Removed licensing FAQs (ahead of relocating).

### CODE REVIEW

File: source/overview/license-and-subscription.rst [hunk 1]
```diff
@@ -172,9 +172,24 @@ Why can't I dismiss the expiry notification banner?
 
 If there is a red expiry announcement banner stating: "Enterprise license is expired and some features may be disabled. Please contact your System Administrator for details." it means your grace period has expired. This announcement banner persists until the license is renewed, and is visible to users.
 
-Once a new license is applied, the banner will no longer be visible. 
+Once a new license is applied, the banner will no longer be visible.
+
+If you don't plan to renew your Enterprise Edition subscription, revoke the expired license in **System Console > Edition and License**.
+
+Do you have a program for official non-profits and charities?
```
Comment (jasonblais) [inline source/overview/license-and-subscription.rst:179]:
```suggestion
Do you have a program for official non-profits, open source projects, and charities?
```

File: source/overview/license-and-subscription.rst [hunk 2]
```diff
@@ -186,14 +201,18 @@ Can other members of my organization use this account to manage our subscription
...
 
-
 What happens if my department buys Mattermost Enterprise Edition and then central IT buys a high volume license that also covers my department?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Mattermost Enterprise Edition subscriptions and support benefits are licensed per production instance.
 
 When the subscription term for your department's production instance expires, you can either discontinue your department's production instance and move to the instance hosted by central IT (which can optionally provision one or more teams for your department to control), or you can renew your subscription to maintain control of your department's instance (e.g., to configure or customize the system in a manner highly specific to your line-of-business) in addition to using the instance from central IT.
 
+Where can I find the license agreement for Mattermost Enterprise Edition?
```
Comment (jasonblais) [inline source/overview/license-and-subscription.rst:211]: Duplicated above

File: source/overview/faq.rst [hunk 3]
```diff
@@ -265,29 +269,157 @@ Working for Mattermost
...
+See our `frequently asked questions about licensing <https://docs.mattermost.com/overview/license-and-subscription.html#frequently-asked-questions>`_.
 
 Do you have discounted licenses for academic institutions?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-See our `frequently asked questions about licensing <https://about.mattermost.com/pricing/#faq>`__.
+See our `frequently asked questions about licensing <https://docs.mattermost.com/overview/license-and-subscription.html#frequently-asked-questions>`_.
+
+Mattermost Source Available License
+-----------------------
```
Comment (jasonblais) [inline source/overview/faq.rst:285]:
```suggestion
----------------------------------------------
```

File: source/overview/faq.rst [hunk 4]
```diff
@@ -265,29 +269,157 @@ Working for Mattermost
...
+
+What is the Mattermost source available icense?
```
Comment (jasonblais) [inline source/overview/faq.rst:287]:
```suggestion
What is the Mattermost source available license?
```

File: source/overview/faq.rst [hunk 5]
```diff
@@ -265,29 +269,157 @@ Working for Mattermost
...
+
+How are repositories changing?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As we add enterprise-only functionality, we will update the license on affected Mattermost authored plugin repositories. The intent is to work alongside the existing, open source functionality in our plugins while reserving certain enterprise functionality to customers who pay us for enterprise licenses.
+
+To which repositories does this apply?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We plan to apply this license to the enterprise directories of our Jira, Microsoft O365 Calendar, and MS Teams Meetings plugins. We also intend to release our Incident Response and Channel Export plugins, currently in beta, under the Mattermost Source Available License. New, Mattermost-authored plugins will generally be released under the Mattermost Source Available License. When we update the licenses, we will release a new version and note the change in the README file of the GitHub repository and as part of any release notes.
```
Comment (jasonblais) [inline source/overview/faq.rst:318]: Do we want to link to the specific repositories, e.g. Jira?

File: source/overview/faq.rst [hunk 6]
```diff
@@ -265,29 +269,157 @@ Working for Mattermost
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We plan to apply this license to the enterprise directories of our Jira, Microsoft O365 Calendar, and MS Teams Meetings plugins. We also intend to release our Incident Response and Channel Export plugins, currently in beta, under the Mattermost Source Available License. New, Mattermost-authored plugins will generally be released under the Mattermost Source Available License. When we update the licenses, we will release a new version and note the change in the README file of the GitHub repository and as part of any release notes. 
+
+We expect to keep plugins without an enterprise component under our open source license. No licensing changes are planned to non-plugin repositories, such as mattermost-server or mattermost-webapp.
+
+Will the repositories be public?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, existing repositories will stay public. We are now also able to make public several enterprise-only plugins previously developed in private.
```
Comment (jasonblais) [inline source/overview/faq.rst:325]:
```suggestion
Yes, existing repositories will stay public. We are now also able to make public several enterprise-only plugins under the Mattermost Source Available License previously developed in private.
```

Comment (jasonblais) [reply source/overview/faq.rst:325]: Just to clarify they are being made public under the source available license, not under open source license.

File: source/overview/faq.rst [hunk 7]
```diff
@@ -265,29 +269,157 @@ Working for Mattermost
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+No, if you have already signed the `Mattermost CLA <https://mattermost.org/mattermost-contributor-agreement/>`_, you do not need to sign it again.
+
+Can I compile your plugins by myself?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes. If you have a Mattermost Enterprise E20 license, you are free to compile and use a plugin under the Mattermost Source Available License. Furthermore, if you are developing against or testing with such a plugin, you are free to compile and test a plugin even without a Mattermost Enterprise E20 license. Without an Enterprise E20 license, source available plugins may have reduced functionality or refuse to start altogether. Request a `trial license <https://mattermost.com/trial/>`_ if your testing requires access to enterprise functionality.
+
+Several of our customers value complete access to our source code and compile our plugins from scratch before deploying to their production servers. By adopting the Mattermost Source Available License, we can develop enterprise-only features in public without impacting this workflow.
```
Comment (jasonblais) [inline source/overview/faq.rst:342]:
```suggestion
Several of our customers value complete access to our source code and compile our plugins from source before deploying to their production servers. By adopting the Mattermost Source Available License, we can develop enterprise-only features in public without impacting this workflow.
```

File: source/overview/faq.rst [hunk 8]
```diff
@@ -265,29 +269,157 @@ Working for Mattermost
...
+
+Will the repositories be public?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, existing repositories will stay public. We are now also able to make public several enterprise-only plugins previously developed in private.
+
+Can I still contribute?
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, we continue to welcome all contributions. Mattermost may select some contributions as enterprise features and license them under the Mattermost Source Available License. As with all Mattermost repositories, you will still need to sign the `Mattermost CLA <https://mattermost.org/mattermost-contributor-agreement/>`_. We will not accept contributions without signing the Mattermost CLA.
```
Comment (jasonblais) [inline source/overview/faq.rst:330]:
```suggestion
Yes, we continue to welcome all contributions. Mattermost may select some contributions as enterprise features and license them under the Mattermost Source Available License. We will aim to communicate such decisions as early as possible in the contribution process.

As with all Mattermost repositories, you will still need to sign the `Mattermost CLA <https://mattermost.org/mattermost-contributor-agreement/>`_. We will not accept contributions without signing the Mattermost CLA.
```

File: source/overview/faq.rst [hunk 9]
```diff
@@ -265,29 +269,157 @@ Working for Mattermost
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+At this time, we have no plans to distribute more than one version of each of our plugins. Without a Mattermost Enterprise E20 License, plugins may have reduced functionality or refuse to start altogether.
+
+Can I continue to use the existing open-source repositories without restriction?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, the Mattermost Source Available License will only apply from the date it is added and to the versions in which it is included.
+
+Can I publish my own plugin under a license of my choosing?
```
Comment (jasonblais) [inline source/overview/faq.rst:354]: This confused me now that I'm reading this question again. At first I thought this meant I can publish my own version of the source available plugin under a license of my choosing. However, I believe this means a different plugin I've built.

I wonder how much value this question adds, @lieut-data?

File: source/overview/faq.rst [hunk 10]
```diff
@@ -265,29 +269,157 @@ Working for Mattermost
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We trust our community to honour the Mattermost Source Available License and work alongside us to develop features across our free and paid offerings. Our support team does not provide support to unlicensed, enterprise-only functionality.
+
+If I make my own plugin using your source available code, can I remove the license restriction?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+No, the Mattermost Source Available License continues to apply to modifications.
+
+Can I write my own plugin and license it under the Mattermost Source Available License?
```
Comment (jasonblais) [inline source/overview/faq.rst:374]: I wonder if this is a better fit in an internal doc for our own team's use?

File: source/overview/faq.rst [hunk 11]
```diff
@@ -265,29 +269,157 @@ Working for Mattermost
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes. 
+
+How do I apply the Mattermost Source Available License?
```
Comment (jasonblais) [inline source/overview/faq.rst:379]: I wonder if this is a better fit in an internal doc for our own team's use?

Comment (lieut-data) [reply source/overview/faq.rst:383]: Agree, @justinegeffen, can we remove this and I can find a home for it elsewhere?

File: source/overview/faq.rst (hunk 10 above)
Comment (lieut-data) [reply source/overview/faq.rst:378]: Let's remove this section altogether as per discussion on community.

File: source/overview/faq.rst [hunk 12]
```diff
@@ -265,29 +271,142 @@ Working for Mattermost
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+At this time, we have no plans to distribute more than one version of each of our plugins. Without a Mattermost Enterprise E20 License, plugins may have reduced functionality or refuse to start altogether.
+
+Can I continue to use the existing open source repositories without restriction?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, the Mattermost Source Available License will only apply from the date it is added and to the versions in which it is included.
+
+Can I publish my own plugin under a license of my choosing?
```
Comment (justinegeffen) [inline source/overview/faq.rst:358]:
```suggestion
Do I need to use the Mattermost Source Available License for plugins I create?
```

File: source/overview/faq.rst [hunk 13]
```diff
@@ -265,29 +271,142 @@ Working for Mattermost
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, you are free to license your own code as you see fit. We will not apply the Mattermost Source Available License either to the `starter-template <https://github.com/mattermost/mattermost-plugin-starter-template>`_ or `demo <https://github.com/mattermost/mattermost-plugin-demo>`_ plugins, leaving them under a permissive open source license to give you the freedom to develop your own plugins.
```
Comment (justinegeffen) [inline source/overview/faq.rst:361]:
```suggestion
You are free to license your own code as you see fit. We will not apply the Mattermost Source Available License either to the `starter-template <https://github.com/mattermost/mattermost-plugin-starter-template>`_ or `demo <https://github.com/mattermost/mattermost-plugin-demo>`_ plugins, leaving them under a permissive open source license to give you the freedom to develop your own plugins.
```

File: source/overview/faq.rst (hunk 9 above)
Comment (justinegeffen) [reply source/overview/faq.rst:358]: I've suggested some rewording.

File: source/overview/faq.rst [hunk 14]
```diff
@@ -265,29 +271,142 @@ Working for Mattermost
...
+
+To which repositories does this apply?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We plan to apply this license to the enterprise directories of our `Jira <https://github.com/mattermost/mattermost-plugin-ee-jira>`_, `Microsoft Calendar <https://github.com/mattermost/mattermost-plugin-mscalendar>`_, and `Microsoft Teams Meetings <https://github.com/mattermost/mattermost-plugin-msteams-meetings>`_ plugins. We also intend to release our `Incident Response <https://github.com/mattermost/mattermost-plugin-incident-response>`_ and `Channel Export <https://github.com/mattermost/mattermost-plugin-channel-export>`_ plugins, currently in beta, under the Mattermost Source Available License. New, Mattermost-authored plugins will generally be released under the Mattermost Source Available License. When we update the licenses, we will release a new version and note the change in the ``README.md`` file of the GitHub repository and as part of any release notes.
```
Comment (amyblais) [inline source/overview/faq.rst:320]: Currently all the plugin repos listed here are private so I don't know if people outside the Mattermost team can access them.

File: source/overview/license-and-subscription.rst [hunk 15]
```diff
@@ -172,9 +172,19 @@ Why can't I dismiss the expiry notification banner?
...
-Once a new license is applied, the banner will no longer be visible. 
+Once a new license is applied, the banner will no longer be visible.
+
+If you don't plan to renew your Enterprise Edition subscription, revoke the expired license in **System Console > Edition and License**.
 
-If you do not plan to renew your Enterprise Edition subscription, revoke the expired license in **System Console > Edition and License**.
+Do you have a program for official non-profits, open source projects, and charities?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, please see the `Mattermost Monday License program <https://mattermost.com/nonprofit/`_.
```
Comment (amyblais) [inline source/overview/license-and-subscription.rst:182]: Link here is broken.

Comment (justinegeffen) [reply source/overview/license-and-subscription.rst]: Fixed, thanks.

File: source/overview/faq.rst (hunk 14 above)
Comment (justinegeffen) [reply source/overview/faq.rst:324]: @jasonblais, @lieut-data: I'm 0/5 on links, let me know what you think.

Comment (lieut-data) [reply source/overview/faq.rst:324]: All of these will be public once the source available license gets applied. This is also a paragraph that's decidedly "set in time" that seems likely to get updated or remove over time, so as long as we're ok revisiting it later, I'm ok with it as written.

**Pull request merged**